lib/misty/http/client.rb

```diff
@@ -67,16 +67,17 @@ def initialize(auth, config, options)
       # and Service API has "/v1/{tenant_id}/stacks"
       # then the path prefix is ignored and path is only "/stacks"
       def self.prefix_path_to_ignore
-        ""
+        ''
```

Comment:
Hello gildub,

I can, of course, replace the double quotes with simple single quotes. But it has less to do with the trend than much more with the performance. While double quotes are evaluated by the ruby interpreter characters inside single quotes are considered as static strings. Therefore, simple quotes are more powerful. This means for example that the string "test_ {1}" is interpreted as 'test_1' and 'test _ # {1}' remains 'test _ # {1}'. But I do not insist on single quotes as there are probably performance improvements in the millisecond range. So I can changed it back to double quotes.